app/models/currency.rb

```diff
@@ -13,6 +13,8 @@
 #
 
 class Currency < ActiveRecord::Base
+  validates :name, uniqueness: { scope: [:code, :country_code, :short_symbol, :symbol] }
```

Comment:
What's this for? Comment maybe?

Reply:
In the specs it was creating like 50+ copies of the same currency, the other changes to the currency on related factories also prevent that, but the validation helps ensure no literally identical currencies get inserted.